packages/contracts-bedrock/test/L1/OptimismPortal.t.sol

```diff
@@ -532,6 +532,8 @@ contract OptimismPortal_Test is CommonTest {
 
     /// @dev Tests that `depositERC20Transaction` reverts when the gas paying token is ether.
     function test_depositERC20Transaction_noCustomGasToken_reverts() external {
+        // TODO(opcm upgrades): remove skip once upgrade path is implemented
+        skipIfForkTest("OptimismPortal_Test: gas paying token functionality DNE on op mainnet");
```

Comment:
Would be nice if `skipIfForkTest` took a chain ID as a parameter

Reply:
Good call, though we'd need to either: 
1. take an array of chain IDs as a parameter. 
2. call `skipIfForkTest()` once per chain ID that its needed on.

Either way it's going to be ugly, so I'd like to defer it until we implement[ multichain CI testing](https://github.com/ethereum-optimism/optimism/issues/13401#issuecomment-2546282590).

Reply:
Sgtm, happy to defer it